sections/kerberos_the_madness.md

```diff
@@ -146,3 +147,7 @@ hence should have their replication factor increased), and which do not get
 used more then 7 days after their creation —and hence can be automatically deleted
 as part of a workflow.
 
+HBase, Hive and HDFS allow for creation and management of such audit logs.  Various 
```

Comment:
We don't really need to repeat the same line from Authorization to convey that Apache Ranger can provide uniform auditing capabilities for HDFS, Hive and HBase.